src/OrchardCore.Modules/Orchard.Tokens.Content/Services/ISlugService.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Orchard.Tokens.Content.Services
+{
+    public interface ISlugService
+    {
+        /// <summary>
+        /// Transforms specified text to the form suitable for URL slugs
+        /// </summary>
+        /// <param name="text">the text to transform</param>
```

Comment:
There will be a day, when I learn all these little details. At least I hope so :-)